### PR TITLE
Building a Cryptocurrency Tracking Telegram Bot with Python

### LINKED ISSUE #1221: Building a Cryptocurrency Tracking Telegram Bot with Python
### Brief Summary: 
A telegram bot is an account on telegram social media powered by a script built to serve as chatbots, games, and automation tools. In this article, we will be going over what they are, how they work, how to build them, then proceed to build a telegram bot that will send updates on cryptocurrency prices to users using Python.

### Key Takeaways:
By the end of this tutorial, the reader will be able to:
* define what telegram bots are
* build a telegram bot with Python
* build scripts to track cryptocurrency prices
* build a telegram bot that regularly sends cryptocurrency price updates

### References:
* https://core.telegram.org/bots
* https://python-telegram-bot.readthedocs.io/en/stable/

### PR DESCRIPTION
**Pull request by LordGhostX**
#### Preliminary Checks - Formatting and Structure 
- [ ] Does your article follow any of the suggested structure formats? - **see [example formats](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md#example-format-structure)**
- [ ] Is your article properly formatted in Markdown? - **see [Github Markdown guide](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet)**
- [ ] Have you used the correct folder and file structure? - **see [Contribution Guidelines](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md)**
- [ ] Is your article over 750 words? - **Tool**: [Wordcounter.net](https://wordcounter.net)
- [ ] Have you used our preferred heading structure? - **[H3 (###) and up](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#headers)**
- [ ] Does your article provide enough value and detail about your topic? - **Articles should be clear, accurate, and fully explained.**
- [ ] Can your article be understood by beginners? - **Assume the audience is smart but has no prior exposure to the common terminology in your article.**
- [ ] Have you included a hero image that is 600x400 pixels and have the copyright to use it? - **Tools:** [Pixlr Image Editor](https://pixlr.com/e) for resizing and [Unsplash](https://unsplash.com) for Creative Commons images
- [ ] In your PR - add a comment that states **["This fixes #(enter your Idea Content Form #)"](https://youtu.be/TKJ4RdhyB5Y?t=278)** to link and close your Idea Content (issue) when the PR is merged.

**NOTE: (1st time contributors - submit ONLY one article to get familiar with the process and style-once published, you can then submit any number of articles.)**
      
#### Grammar Checks
- [ ] Have you spellchecked and grammar checked your article? - **Tools:** [Write&Improve](https://writeandimprove.com/?lang=en-GB) - [ProWritingAid](https://prowritingaid.com/) -  [Grammarly](https://grammarly.com) 
- [ ] Have you checked your article for readability? - **Tool:** [Hemmingway](http://www.hemingwayapp.com)
- [ ] Have you added sources for quotes and images that aren't yours?

#### Technical Checks
- [ ] Are your code snippets properly formatted for syntax highlighting - **see [Syntax guide](https://gohugo.io/content-management/syntax-highlighting)**

### Contribution Guidelines
For first-time contributors and for more details, **see [Contributing Guidelines](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md)**

### CODE REVIEW

**Comment by lalith1403** [comment]
This fixes #1221

File: articles/cryptocurrency-tracking-telegram-bot/index.md [hunk 1]
```diff
@@ -0,0 +1,189 @@
...
+
+![](/engineering-education/cryptocurrency-tracking-telegram-bot/2en1wn0m-8gcycfjmtvs.png)
+
+## Building a Telegram Bot
+
+#### Step 1: Have a Conversation with BotFather
+
+Sign In to your Telegram account, then search for `@botfather` and start a conversation with the account. PS: BotFather is also a Telegram bot.
+
+![](/engineering-education/cryptocurrency-tracking-telegram-bot/2frh-rcjna_zv5g1epn7.png)
```
Comment (lalith1403) [inline articles/cryptocurrency-tracking-telegram-bot/index.md:75]: alt text here. This is the format for inserting an image. Can you insert texts describing the image in 2-3 words in the space allotted for it? Do it for all the images.

**Review by lalith1403 — APPROVED** [review]
Add the tags and we can proceed with the final round of review. It was a fun project. Congratulations on your first article :)

File: articles/cryptocurrency-tracking-telegram-bot/index.md (hunk 1 above)
Comment (LordGhostX) [reply articles/cryptocurrency-tracking-telegram-bot/index.md]: Sure no problem, I've added the alt tag to all images

**Review by lalith1403 — APPROVED** [review]

**Comment by hectorkambow** [comment]
👋 Hector here - and welcome to the EngEd program - thank you for your article submission. 

I will be doing a final review when the PR is ready. 

Upon running this article through our 3rd party plagiarism checker it raised a few red flags (please see attached). 

Please revisit the article to bring down the % - take in mind that we strive to have 10% or lower before publishing and be sure to include any extra sources if any are missing.

We want to make sure all our EngEd content is original and unique - and this means making sure all our content is plagiarism free. 

Be sure to check our our resources page ([the link on plagiarism](https://www.scribbr.com/category/plagiarism/)) to go over any misconceptions.

**Comment by LordGhostX** [comment]
@hectorkambow @lalith1403 I've updated my article and got 0% plagiarism on http://plagiarismdetector.net/ https://smallseotools.com/plagiarism-checker/ and https://www.duplichecker.com

**Comment by lalith1403** [comment]
@hectorkambow This is one of the articles with plagiarism issues.

**Comment by hectorkambow** [comment]
@lalith1403  
Plagiarism check done 👍

**Pull request merged**